### PR TITLE
fix data

### PR DESCRIPTION
**Pull request by Jo-Zh**
just retrieve the latest data and updated question-data
to test:
'yarn test'

### CODE REVIEW

File: tests/components/MCQOption.test.ts
Comment (anzau) [inline tests/components/MCQOption.test.ts:1]: We definitely need to look at updating this test - We can't be reliant on data like this.

File: data/question-data.json [hunk 1]
```diff
@@ -1,5954 +1,7473 @@
...
+              "subject": "Physiology",
+              "system": "Neurophysiology"
+          },
+          "statement": "<p>Which part of a neuron receives information from surrounding cells?</p>",
+          "optionsList": [
+              {
+                  "type": 1,
+                  "optionValue": "<p>Axon</p>",
+                  "optionCorrect": false,
+                  "currentState": ""
```
Comment (anzau) [inline data/question-data.json]: Why do we have currentState?

File: data/question-data.json [hunk 2]
```diff
@@ -1,5954 +1,7473 @@
...
+      {
+          "tags": {
+              "course": "VETS2011",
+              "subject": "Physiology",
+              "system": "Neurophysiology"
+          },
+          "statement": "<p>Which part of a neuron receives information from surrounding cells?</p>",
+          "optionsList": [
+              {
+                  "type": 1,
```
Comment (anzau) [inline data/question-data.json]: We don't need the type here...

File: tests/components/MCQOption.test.ts
Comment (Jo-Zh) [reply tests/components/MCQOption.test.ts:1]: fixed

File: data/question-data.json (hunk 1 above)
Comment (Jo-Zh) [reply data/question-data.json]: fixed

**Review by anzau — APPROVED** [review]

**Pull request merged**